maquetado/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
```

Comment:
This is only a comment normally we use color on HEX nomenclature, I know training say that use color : red but for coming prs use Hex